keyboards/signum31/config.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xF1E7
+#define PRODUCT_ID      0x5163
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    troyfletcher
+#define PRODUCT         keyboard
+#define DESCRIPTION     Keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 10
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { D2, D1, F5, B5, B2, C6 }
+#define MATRIX_COL_PINS { B4, D7, D0, E6, D4, F6, F4, F7, B1, B3 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* number of backlight levels */
+
+#ifdef BACKLIGHT_PIN
+#define BACKLIGHT_LEVELS 0
+#endif
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING_DELAY 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
```

Comment:
Could this be removed?

Reply:
Probably yes. I guess you mean till line 43?

I didn't write this code myself. Troy provided a .json file for "https://kbfirmware.com/".
I imported the file on that website, downloaded the .hex from there and then started editing it manually from that point onwards. All code that I didn't write myself might be autogenerated from that website, but even if it was provided indirectly from Troy via the .json file, it is at least 3 months old and might be deprecated by now.